UIKitDZ/11_UITabBarController/11_UITabBarController/TimerViewController.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  TimerViewController.swift
+//  11_UITabBarController
+//
+//  Created by admin on 02.07.2021.
+//
+
+import UIKit
+  
+final class TimerViewController: UIViewController, UIPickerViewDelegate {
+
+    @IBOutlet weak var pickerView: UIPickerView!
+    
+    
+     private var hour: Int = 0
+     private var minutes: Int = 0
+     private var second: Int = 0
```

Comment:
тут тип лучше не указывать
```suggestion
     private var second = 0
```

Reply:
убрал типы